### ISSUE TITLE
bug: RangeError: Invalid count value

### ISSUE THREAD

**Issue by tdeekens** [checkly/checkly-cli#697]
### Node.js version

19

### NPM version

9

### @checkly/cli version

4.03

### Steps to reproduce

Trigger checks using `--trigger` while passing some `--tags`. It does not fail on every run. It's not clear what happens inside the CLI causing this.

The output on CI is

```bash
RangeError: Invalid count value: -4
    at String.repeat (<anonymous>)
    at formatSectionTitle (/home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/reporters/util.ts:234:39)
    at formatCheckResult (/home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/reporters/util.ts:98:7)
    at CiReporter.onCheckEnd (/home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/reporters/ci.ts:29:45)
    at /home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/commands/trigger.ts:130:32
    at Array.forEach (<anonymous>)
    at TriggerRunner.<anonymous> (/home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/commands/trigger.ts:130:17)
    at TriggerRunner.emit (node:events:512:28)
    at TriggerRunner.emit (node:domain:489:12)
    at TriggerRunner.processMessage (/home/circleci/.npm/_npx/a66ee913fe899ce0/node_modules/checkly/src/services/abstract-check-runner.ts:154:12)
```

### What is expected?

The checks to fail or succeed without running into an RangeError.

### What is actually happening?

Checks run into a RangeError.

**Comment by umutuzgur** [comment]
@clample could you take a look at this one when you have the availability? It is a weird error

**Comment by clample** [comment]
@tdeekens thank you for reporting this issue. This should fixed in version 4.0.6 of the CLI.

Some CI environments set a terminal width of 0. The CLI wasn't handling this correctly, and I believe that was causing this RangeError when the CLI formatted output for the terminal. There should be better handling for this case now.
